public-api.txt

```diff
@@ -0,0 +1,384 @@
+pub async fn fastcrypto::SignatureService::request_signature(&mut self, digest: Digest) -> Signature
```

Comment:
Is this auto-generated? If so how/by what?

Reply:
using `cargo public-api`, worth adding to the ci but it needs an initial publick-api.txt and ci can detect changes

https://github.com/Enselic/cargo-public-api/tree/main